Documentation/Settings.cfg

```diff
@@ -3,25 +3,25 @@
 
 [general]
 
-project     = <project>
-version     = <version>
-release     = <release>
-copyright   = <copyright>
+project     = Content and Marketing Guide
+version     = 1.0
+release     = 1.0
+copyright   = TYPO3 Project Contributors
 
 [html_theme_options]
 
 # "Edit on GitHub" button
-github_repository    = <github-repository>
-github_branch        = <github-branch>
+github_repository    = https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
```

Comment:
Missed this one: The repository must not include the protocol and domain, but only 
```
github_repository = TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
```
See [here](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#github-workflow) for further details.